cmsplugin_articles_ai/factories.py

```diff
@@ -47,6 +55,7 @@ class Meta:
     title = factory.Faker("catch_phrase")
     slug = factory.LazyAttribute(lambda article: slugify(article.title))
     published_from = timezone.now() - timedelta(days=1)
+    category = factory.Iterator(Category.objects.all())
```

Comment:
Not sure if I like this approach since this seems to cause the need for repeating `CategoryFactory()` in multiple places which isn't nice. I would keep the existing tests as is i.e. not requiring categories since this plugin needs to work also without categories IMO (as it seems to do in this PR). But keeping the existing tests as is would make sure of that.

Some ideas of getting rid of the category requirement in this factory:
- Add some checking so that category is assigned only if categories exist
- Make another factory that assigns also the category and keep category as null in here